docs/src/architecture/08_concepts/document_templates/proposal/F14-Generic/0ce8ab38-9258-4fbc-a62e-7faa6e58318f.schema.json

```
@@ -489,6 +490,172 @@
                 "open_source"
             ]
         },
+        "campaign_category": {
+            "type": "object",
+            "title": "Campaign Category",
+            "description": "Determine the eligibility of the proposal for this category",
+            "properties": {
+                "category_questions": {
+                    "type": "object",
```

Under `segment` there should be a section:

```suggestion
                    "$ref": "#/definitions/section",
```